src/style.css

```diff
@@ -0,0 +1,159 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
use * selector only for box-sizing. For other properties indicate which tags need it

Reply:
Ok. But what is better to reset m0, p0 for tags in general or for selected classes?